### ISSUE TITLE
The prompt fills the screen if the search result line is too long

### ISSUE THREAD

**Issue by casr** [jhawthorn/fzy#55]
Sorry I thought #54 fixed this but I have just found another instance where this occurs with or without scores.

Test case:
```sh
cd /path/to/src/fzy
git reset --hard 0ee47c2
make clean ; make
ag --nogroup '^' | ./fzy
```
and at the prompt do type `jjj` and then repeatedly switch between having `jj` and `jjj`. The prompt soon fills up the screen.

Related to #47.

**Comment by casr** [comment]
I should probably add that `echo $COLUMNS` results in `84`.

**Comment by neuschaefer** [comment]
I think this happens when a result overflows its line because of tab characters. Tabs are currently counted as one screen cell, like any other character.

**Comment by casr** [comment]
Appears to be fixed for me since 757b31309434d7adcd1361be26c1cd2a44d6ec9b
